third_party/nlbuild-autotools/repo/scripts/mkversion

```diff
@@ -161,7 +161,11 @@ gitversion() {
 				fi
 
 			else
-				gitversion=`${PRINTF} "g${head}"`
```

Comment:
this should go to third_party project commit directly
OT just pulls in third party code from their master repos